packages/components/form/src/label.ts

```diff
@@ -138,37 +137,8 @@ export class Label extends LitElement {
     }
   }
 
-  async #update(): Promise<void> {
-    // Give the component & siblings time to set the required attribute
-    await this.updateComplete;
-
-    const { form } = this.formControl || {},
-      required = this.formControl?.hasAttribute('required');
-
-    this.disabled = this.formControl?.hasAttribute('disabled') ?? false;
-
-    if (form) {
-      const controls = Array.from(form.elements).filter(el => el.tagName !== 'SL-BUTTON');
-
-      // Count the required form controls
-      const requiredCount = controls.reduce((count, control) => {
-        return count + (control.hasAttribute('required') ? 1 : 0);
-      }, 0);
-
-      /**
-       * If the required form controls outnumber the optional form controls,
-       * then mark the optional form controls. If the optional form controls
-       * outnumber the required form controls, mark the required form controls.
-       * If there is only a single form element, do nothing.
-       */
-      const optionalCount = controls.length - requiredCount,
-        showRequired = requiredCount <= optionalCount;
-
-      this.optional = !required && !showRequired;
-      this.required = required && showRequired;
-    } else {
-      this.optional = false;
-      this.required = required;
-    }
+  #update(): void {
+    this.disabled = this.formControl?.disabled ?? false;
+    this.required = this.formControl?.required ?? false;
   }
 }
```

Comment:
It looks like those label stories for optional/required needs to be updated? :)

<img width="1260" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/f6c2eae2-c4b5-4fee-b3fa-f182cc4a82c8">

Reply:
Fixed!